apps/vscode/command_client/command_client.py

```diff
@@ -138,142 +52,15 @@ def run_command(
     # variable argument lists
     args = [x for x in args if x is not NotSet]
 
-    communication_dir_path = get_communication_dir_path()
-
-    if not communication_dir_path.exists():
-        if args or return_command_output:
-            raise Exception("Must use command-server extension for advanced commands")
-        raise NoFileServerException("Communication directory not found")
-
-    request_path = communication_dir_path / "request.json"
-    response_path = communication_dir_path / "response.json"
-
-    # Generate uuid that will be mirrored back to us by command server for
-    # sanity checking
-    uuid = str(uuid4())
-
-    request = Request(
-        command_id=command_id,
-        args=args,
-        wait_for_finish=wait_for_finish,
-        return_command_output=return_command_output,
-        uuid=uuid,
+    return actions.user.rpc_client_run_command(
+        actions.user.command_server_directory(),
+        actions.user.trigger_command_server_command_execution,
+        command_id,
+        args,
+        wait_for_finish,
+        return_command_output,
```

Comment:
We're starting to build up a bunch of parameters on this action and I'm worried about the wrong ones getting passed in the future; how do you feel about converting these to kwargs to make them explicit? It's much more verbose but I think it's worth the tradeoff in these situations

```suggestion
    return actions.user.rpc_client_run_command(
        dir_name=actions.user.command_server_directory(),
        trigger_command_execution=actions.user.trigger_command_server_command_execution,
        command_id=command_id,
        args=args,
        wait_for_finish=wait_for_finish,
        return_command_output=return_command_output,
```

Reply:
I don't think you can use named arguments with Talon actions.